.github/PULL_REQUEST_TEMPLATE/RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+> ### Before creating a release PR
```

Comment:
We want the entire checklist, even all the stuff that doesn't have to do with creating the PR specifically.

Also I'm not sure we want to have the checklist quoted with `>`?

Reply:
My bad, I quoted the original to copy and forgot to remove it :)

Reply:
Let's name this file `release.md`